packages/react-storage/src/components/StorageBrowser/views/LocationActionView/hooks/types.ts

```diff
@@ -0,0 +1,11 @@
+import { ControlsContext } from '../../../controls/types';
+
+export interface UseActionView {
+  controlsContextValue: ControlsContext;
```

Comment:
This shouldn't be exposed on any variant of `useView`, why do you need it?

Reply:
I can move it down into the view, was trying to encapsulate everything into the hook

Reply:
Moved down to view, exposed the taskCounts and tableData instead.